### DISCUSSION TITLE
Raspberry Pico support file

### DISCUSSION THREAD

**Discussion by ThinusBotes1** [revoxhere/duino-coin#1101] [Q&A]
Hi all

How do I get to work on the RPi Pico to get it to work? I have a bunch of them, and I would like to use them all in this project. 

I am looking at using the Micro-python code for this one. I don't really want to use C on the pico.

Thank you in advance

**Comment by kosa6661987** [comment]
Same issue for me, cannot figure out how to fix it.
Upload To board, looks ok, then start up AVR on Pc, sending job to board but no respond, the same issue when run AVR on raspberry pi, no respond from Pico.
